src/lib/ZoomControls.js

```diff
@@ -0,0 +1,134 @@
+import isFinite from 'lodash/isFinite';
+import EventEmitter from 'events';
+import { ICON_ZOOM_IN, ICON_ZOOM_OUT } from './icons/icons';
+
+const CLASS_ZOOM_CURRENT_SCALE = 'bp-zoom-current-scale';
+const CLASS_ZOOM_IN_BUTTON = 'bp-zoom-in-btn';
+const CLASS_ZOOM_OUT_BUTTON = 'bp-zoom-out-btn';
+
+class ZoomControls extends EventEmitter {
```

Comment:
I tried to pattern it off of `PageControls.js`. I don't think we want to extend `Controls.js` because it's more of a composition relationship than an inheritance relationship since we leverage controls.js to manage the buttons